### PR TITLE
Cuda 11.x

### PR DESCRIPTION
**Pull request by IvanKurnosov**

### CODE REVIEW

File: embedding-calculator/Makefile [hunk 1]
```diff
@@ -2,13 +2,14 @@ SHELL := /bin/bash
 .DEFAULT_GOAL := default
```
Comment (pospielov) [inline embedding-calculator/Makefile:2]: It looks like in this file lots of temporary code, please revert it

File: embedding-calculator/Dockerfile [hunk 2]
```diff
@@ -44,9 +44,11 @@ ARG SKIP_TESTS
 COPY pytest.ini .
 RUN if [ -z $SKIP_TESTS  ]; then pytest -m "not performance" /app/ml/src; fi
 
+RUN rm -rf /usr/lib/x86_64-linux-gnu/libcuda.so.1
```
Comment (pospielov) [inline embedding-calculator/Dockerfile:47]: Why do we need this line?

**Review by pospielov — CHANGES_REQUESTED** [review]

File: embedding-calculator/Makefile (hunk 1 above)
Comment (IvanKurnosov) [reply embedding-calculator/Makefile:2]: Made it the same as it is in 1.0.x

File: embedding-calculator/Dockerfile (hunk 2 above)
Comment (IvanKurnosov) [reply embedding-calculator/Dockerfile]: deleted

**Review by pospielov — APPROVED** [review]

**Pull request merged**